### PR TITLE
insight spendables as dict, BatchService parent

### PR DESCRIPTION
**Pull request by tomholub**

### CODE REVIEW

File: multisigcore/providers/insight.py [hunk 1]
```diff
@@ -23,11 +26,13 @@ def spendables_for_addresses(self, bitcoin_addresses):
             addresses = bitcoin_addresses[i:i+CHUNK_SIZE]
             url = "%s/api/addrs/%s/utxo" % (self.base_url, ",".join(addresses))
             r.extend(json.loads(urlopen(url).read().decode("utf8")))
-        spendables = []
+        spendables = {}
```
Comment (devrandom) [inline multisigcore/providers/insight.py:29]: I would like to stay compatible with pycoin for now, and this is an incompatible change.  The address can be calculated from the spendable's script.

Comment (tomholub) [inline multisigcore/providers/insight.py:29]: I see. Could you make that change in multisig-core? As written, it was failing on send tx.

You're right it's better to stay compatible, Richard might end up merging the function back into pycoin.

Comment (devrandom) [inline multisigcore/providers/insight.py:29]: oh, you are right, I didn't realize that I introduced the incompatibility.  Will fix.

Comment (tomholub) [inline multisigcore/providers/insight.py:29]: kk great. Also, notice the `class InsightBatchService(InsightService, BatchService)`, without it, it was still pulling one by one (in my case, it took about 20 seconds :) )

**Pull request closed without merging**